doc/run_benchmark.rst

```diff
@@ -133,7 +133,7 @@ See also the :ref:`check command <check_cmd>`.
 
 .. _stable_bench:
 
-How to get reproductible benchmark results
+How to get reproducible benchmark results
 ==========================================
```

Comment:
The line should have the same length than the title, otherwise Sphinx emits a warning.